health/README.md

```diff
@@ -0,0 +1,11 @@
+health
+====
+
+A simple framework for implementing an HTTP health check endpoint on servers.
+
+Users implement their ```health.Checkable``` types, and create a ```health.Checker```, from which they can get an ```http.HandlerFunc``` using ```health.Checker.MakeHealthHandlerFunc```.
```

Comment:
For inline highlights in markdown you can just use single backticks. Triple backticks is for multi-line code blocks.